configuration/analyzer_config.json

```diff
@@ -994,6 +994,22 @@
     "python_module": "tor.Tor",
     "soft_time_limit": 30
   },
+  "Triage_Search": {
+    "type": "observable",
+    "observable_supported": [
+      "url",
+      "hash"
+    ],
+    "description": "analyze a url sample using triage sandbox environment",
+    "python_module": "triage_search.TriageSearch",
+    "requires_configuration": true,
+    "additional_config_params": {
+      "api_key_name": "TRIAGE_KEY",
+      "endpoint": "public",
+      "report_type": "overview"
```

Comment:
should add `analysis_type: "submit"` parameter. Explicit is better than implicit.